.github/workflows/elixir.yml

```diff
@@ -127,9 +127,17 @@ jobs:
         image: amazon/dynamodb-local:2.0.0
         ports:
           - 8000:8000
+      mosquitto:
+        image: eclipse-mosquitto:2.0
+        ports:
+          - 1883:1883
+        volumes:
+          - ./mosquitto:/mosquitto/config/
+        options: --name mqtt
 
     steps:
       - uses: actions/checkout@v3
+      - run: docker restart mqtt
```

Comment:
question: can you say more about this?

Reply:
When GitHub actions initializes the service, the runner is bare, so we can't mount the config file. This is a workaround from https://github.com/orgs/community/discussions/42127#discussioncomment-7591609.

Reply:
Added a `name` explaining this step.